src/test/java/com/rabbitmq/client/test/functional/MicrometerObservationCollectorMetrics.java

```diff
@@ -111,6 +120,17 @@ private abstract static class IntegrationTest extends SampleTestRunner {
     public TracingSetup[] getTracingSetup() {
       return new TracingSetup[] {TracingSetup.IN_MEMORY_BRAVE, TracingSetup.ZIPKIN_BRAVE};
     }
+
+    void runWithNullingObservation(ObservationRegistry registry, Tracer tracer, Observation.CheckedRunnable<?> runnable) {
+      Observation noParentObservation = Observation.createNotStarted("null_observation", registry);
```

Comment:
With Micrometer 1.10.8 I think we can change this whole method to become `new NullObservation(registry).observeChecked(runnable)`